src/OrchardCore.Modules/OrchardCore.Sitemaps/Startup.cs

```diff
@@ -80,6 +81,13 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddContentPart<SitemapPart>()
                 .UseDisplayDriver<SitemapPartDisplayDriver>()
                 .AddHandler<SitemapPartHandler>();
+
+            //custom url in sitemap
```

Comment:
```suggestion
            // Custom sitemap path.
```

For infos comments are //spaceCapitalLetterFullstop